hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailPlugin.java

```diff
@@ -302,9 +302,9 @@ protected Message createMimeMessage(ActionMessage msg) throws Exception {
             if (alert.getStatus().equals(Status.OPEN)) {
                 email.setSentDate(new Date(alert.getCtime()));
             } else if (alert.getStatus().equals(Status.ACKNOWLEDGED)) {
-                email.setSentDate(new Date(alert.getAckTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             } else {
-                email.setSentDate(new Date(alert.getResolvedTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             }
```

Comment:
I like the getting rid of the redundant fields but it may make sense to keep some convenience getters like:
```
  alert.getOpenTime
  alert.getAckTime
  alert.getResolvedTime
  alert.getStatusTime( status )
```
These convenience methods would reverse-traverse the lifecycle list and return the most recent time for the given status. What do you think?    We could potentially have similar methods that return the whole LifeCycle.  I think it's important to offer an easy way to get to the ackTime even if the current status is Resolved.